freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_install.sh

```diff
@@ -34,6 +34,7 @@ ipa-server-install \
           --no-dnssec-validation \
 {%- endif %}
           --unattended \
+          --dirsrv-config-file /opt/salt/initial-ldap-conf.ldif \
```

Comment:
can this mechanism also be used for setting the nsslapd-sizelimit instead of running a separate script?

Reply:
it's possible, I haven't validated yet

Reply:
it works, I'll refactor my pr once this merged